cocos/platform/desktop/CCGLViewImpl-desktop.cpp

```diff
@@ -39,6 +39,9 @@ THE SOFTWARE.
 #include "base/ccUtils.h"
 #include "base/ccUTF8.h"
 #include "2d/CCCamera.h"
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32) || (CC_TARGET_PLATFORM == CC_PLATFORM_LINUX)
```

Comment:
I think better to use one simple define instead of complex.
Define it once at beginning:
```
#if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32) || (CC_TARGET_PLATFORM == CC_PLATFORM_LINUX)
   #define SET_ICON_SUPPORTED
#endif
```
Later use:
`#ifdef SET_ICON_SUPPORTED`

Pros: in future will be easier to change it, probably for MacOS platform.

Reply:
Done (`CC_ICON_SET_SUPPORT`).